ko/sql/jsp.rst

```diff
@@ -291,9 +297,40 @@ CUBRID 데이터베이스에 Phone 클래스를 생성한다.
 서버 내부 JDBC 드라이버 사용
 ============================
 
-Java 저장 함수/프로시저에서 데이터베이스에 접근하기 위해서는 서버 측 JDBC 드라이버(Server-Side JDBC Driver)를 사용해야 한다. Java 저장 함수/프로시저가 데이터베이스 내에서 실행되기 때문에 서버 측 JDBC 드라이버는 다시 연결을 설정할 필요가 없다.
+Java 저장 함수/프로시저에서 데이터베이스에 접근하기 위해서는 서버 측 JDBC 드라이버(Server-Side JDBC Driver)를 사용해야 한다.
+서버 측 JDBC 드라이버로 다음의 기능이 가능하다.
+
+*    **질의문 수행**
```

Comment:
```suggestion
*    **\질의문 수행**
```

Reply:
What is the effect of the backslash? It looks the same to me with and without it.

Reply:
I also checked and backslush is not needed.